sources/engine/Stride.Graphics/FastTextRenderer.cs

```diff
@@ -247,6 +253,97 @@ public unsafe void End([NotNull] GraphicsContext graphicsContext)
             graphicsContext.CommandList.DrawIndexed(charsToRenderCount * 6);
         }
 
+        public unsafe void GraphicsFastTextRendererGenerateVertices(RectangleF constantInfos, RectangleF renderInfos, string textPointer, ref int textLength, IntPtr vertexBufferPointer)
+        {
+            var vertexBuffer = (VertexPositionNormalTexture**)vertexBufferPointer;
```

Comment:
Can you use `VertexPositionNormalTexture**` as the parameter type instead of `IntPtr`, it'll be clearer for the caller and you can move the type conversion outise the loop, it's likely already inlined by the jit already but might as well.

Reply:
This is actually incorrect as written (and crashes when enabling the Profiler for example). It should just be `VertexPositionNormalTexture*` as I see it.
I suggest something like
`private int GraphicsFastTextRendererGenerateVertices(RectangleF constantInfos, RectangleF renderInfos, string text, Span<VertexPositionNormalTexture> vertexBuffer)` or maybe
`private int GraphicsFastTextRendererGenerateVertices(RectangleF constantInfos, RectangleF renderInfos, string text, ref VertexPositionNormalTexture vertexBuffer)` and iterating with `vertexBuffer = ref Unsafe.Add(ref vertexBuffer, 1)` instead of pointer arithmetic if that's simpler/faster than `Span`.